### PR TITLE
add simplecov for code coverage reports

### PR DESCRIPTION
**Pull request by ricardochimal**
* setting `CI` env var configures simplecov for continuous
  integration output
* setting `COVERAGE` env var generates a report when running tests
  locally`

### CODE REVIEW

File: lib/template/spec/spec_helper.rb [hunk 1]
```diff
@@ -10,6 +10,20 @@
 require "bundler"
 Bundler.require(:default, :test)
 
+if ENV["COVERAGE"] || ENV["CI"]
```
Comment (brandur) [inline lib/template/spec/spec_helper.rb]: Mind just adding a short comment here to the same effect as what your pull request description contained? It should help remind us how these get used.

**Comment by brandur** [comment]
@ricardochimal Minor comment request, but great idea. Thanks!

**Comment by ricardochimal** [comment]
@brandur done!

**Comment by brandur** [comment]
@ricardochimal Thanks!

**Comment by pedro** [comment]
+1, good stuff!

**Pull request merged**